docs/REST API/rest-api.md

```diff
@@ -4276,6 +4276,8 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ### Get host's software
 
+> The new keys/values added in the app management features are in beta and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L4304-L4311).
```

Comment:
```suggestion
> The new keys/values added in the software management features are in beta and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L4304-L4311).
```

Reply:
@noahtalerman I think we don't use "app management" anywhere else. We always use "software" except for App Store apps.

Reply:
Hey @marko-lisica good catch.

We're starting to use "App management" on the website.

Check out [fleetdm.com/device-management](https://fleetdm.com/device-management):
![Screenshot 2024-07-24 at 3 02 06 PM](https://github.com/user-attachments/assets/04d95ac4-d274-4f01-aa02-a5bd87870238)